examples/Db2/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Terraform Module to install and configure DB2 on Openshift
+
+**Module Source**: `github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/Db2`
+
+## Set up access to IBM Cloud
+
+If running these modules from your local terminal, you need to set the credentials to access IBM Cloud.
+
+Go [here](../../CREDENTIALS.md) for details.
+
+### Download required license files
+
+Download required license files from [IBM Internal Software Download](https://w3-03.ibm.com/software/xl/download/ticket.wss) or [IBM Passport Advantage](https://www.ibm.com/software/passportadvantage/) into the  `./files` folder
+```bash
+DB2:
+Part Number : CNB21ML
+Filename    : DB2_AWSE_Restricted_Activation_11.5.zip
+```
+Note: the license key is required only for Advanced DB2 installation
+
+
+
+## Provisioning this module in a Terraform Script
+
+See the example [here](../../examples/Db2) on how to provision this module.
```

Comment:
This just points to itself.  Need to include a section with an example of what to define in the terraform.tfvars file.

Reply:
Done.